docs/interpreter/jdbc.md

```diff
@@ -507,8 +507,9 @@ Here are some examples you can refer to. Including the below connectors, you can
     <td>hive_password</td>
   </tr>
   <tr>
-    <td>hive.proxy.user</td>
-    <td>true or false</td>
+    <td>default.proxy.user.property</td>
+    <td>hive.server2.proxy.user</td>
+  </tr>
 </table>
 
 Connection to Hive JDBC with a proxy user can be disabled with `hive.proxy.user` property (set to true by default)
```

Comment:
I guess the preference would be to have a way to migrate existing users and config by having it automatically converted, add a log warning etc.

but it's your call in the specific case